src/main/java/Parser/Parser.java

```diff
@@ -52,10 +52,11 @@ protected void setDoneTask(String command) throws DukeException {
             throw new DukeException("Oh no! This can't be DONE! (The description of done can't be empty!)");
         } else {
             try {
-                int i = parseInt(command.split(" ")[1]);
-                storage.setDoneLine(i);
-                String doneTask = storage.printLine(i);
+                int index = parseInt(command.split(" ")[1]);
+                storage.setDoneLine(index);
+                String doneTask = storage.printLine(index);
```

Comment:
Maybe certain parts of the class can be extracted out because it is a bit too long.